### PR TITLE
✨ PRTL-894 cases table arrange columns

### PR DESCRIPTION
**Pull request by alex-wilmer**

### CODE REVIEW

File: src/packages/@ncigdc/tableModels/utils.js [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+) => {
+  let idFilter; // because of weird ternary bug https://github.com/babel/babel/issues/5809
+
+  if (type === "case") {
+    idFilter = node => ({
+      field: "cases.project.project_id",
+      value: node.project_id
+    });
+  } else {
+    idFilter = node => ({
```
Comment (cheapsteak) [inline src/packages/@ncigdc/tableModels/utils.js]: Can we pass this in from the call location instead of internalizing it into the function?

Comment (cheapsteak) [inline src/packages/@ncigdc/tableModels/utils.js]: On second glance I would propose we add 2 additional options that take in functions

```
createDataCategorySubColumns({
  ...
  getCellLinkFilters: (rows, category) => makeFilter(...)
  getTotalLinkFilters: (rows, category) => makeFilter(...)
});
```

what do you think?

**Comment by cheapsteak** [comment]
Not sure if this is a concern, but the order of the cases are different

relay: http://deploybot.gdc.cheapsteak.net:33068/repository
image


this pr: http://deploybot.gdc.cheapsteak.net:33060/repository
image

Have not set any sort order or filters

**Comment by cheapsteak** [comment]
Relay shows:
"Available Files per Data Category"

This PR has
"Available Cases per Data Category"

**Comment by cheapsteak** [comment]
Sort order probably doesn't matter, the number of results is the same, as is the filesize

Let's change "Available Cases" to "Available Files" and merge this

**Comment by alex-wilmer** [comment]
@cheapsteak thanks. going to update the util function to have all that stuff passed in from caller

**Comment by alex-wilmer** [comment]
@cheapsteak updated

**Pull request merged**